website/docs/index.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "tfe"
+page_title: "Provider: Terraform Enterprise"
+sidebar_current: "docs-tfe-index"
+description: |-
+  The Terraform Enterprise provider is used to interact with the many resources supported by (Private) Terraform Enterprise. The provider needs to be configured with the proper credentials before it can be used.
+---
+
+# Terraform Enterprise Provider
+
+The Terraform Enterprise provider is used to interact with the many resources
+supported by (Private) [Terraform Enterprise](https://www.hashicorp.com/products/terraform).
```

Comment:
if this works with SaaS TFE - should we ditch `Private` here and above?

Reply:
Well it supports both SaaS TFE and private TFE installs... So I think it makes sense to write that in the docs.